cpp/dolfinx/fem/discreteoperators.h

```diff
@@ -107,21 +109,23 @@ void discrete_gradient(const FunctionSpace& V0, const FunctionSpace& V1,
   assert(dofmap1);
 
   // Build the element interpolation matrix
-  std::vector<T> A(e1->space_dimension() * ndofs0);
+  std::vector<T> Ab(e1->space_dimension() * ndofs0);
```

Comment:
I think you have used `*_b` on most of the storages, so we should change this if we want to be consistent.

Reply:
Pattern is that `_b` is added only when the data is then wrapped using `mdspan`.